app/templates/direct-award/index.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block page_title %}Saved searches - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('main.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": "/buyers",
+          "label": "Account Overview"
+      },
+      {
+          "label": "Saved searches"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="direct-award-saved-searches">
+  <header class="page-heading">
+    <h1>
+      Saved searches
+    </h1>
+    <p class="context">Find, compare and buy cloud services through the G-Cloud framework.</p>
+  </header>
+
+  <div class="grid">
+    
+    {{ summary.heading("Searching", id="searching_table") }}
```

Comment:
'Current searches' perhaps? We can leave the final decisions for @karlchillmaid but we can tweak in the meantime :)